ros2_control/package.xml

```diff
@@ -15,6 +15,8 @@
   <exec_depend>controller_manager</exec_depend>
   <exec_depend>controller_manager_msgs</exec_depend>
   <exec_depend>hardware_interface</exec_depend>
+  <exec_depend>joint_limits</exec_depend>
+  <exec_depend>ros2controlcli</exec_depend>
```

Comment:
Do you still need this? Doesn't seem to fit with the rest of the PR.

Reply:
```suggestion
```